lib/components/Charts/AreaChart/index.tsx

```diff
@@ -21,11 +21,28 @@ import { fixedForwardRef } from "../utils/forwardRef"
 import { prepareData } from "../utils/muncher"
 import { ChartConfig, ChartPropsBase, InferChartKeys } from "../utils/types"
 
+type allowedLineTypes =
```

Comment:
We probably don’t need all of them, right? Not entirely sure what the monotoneX one does, but it seems to get the job done. Maybe we can stick with that and simplify the options?
(This would be a great topic to discuss with Desi).

Reply:
[remove extra area chart line charts](https://github.com/factorialco/factorial-one/pull/449/commits/33c7ed9153eb2c955673b76044dcf202484c3d95)